ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/application-traffic-recognition.j2

```diff
@@ -0,0 +1,138 @@
+{#
+    Copyright (c) 2023 Arista Networks, Inc.
+    Use of this source code is governed by the Apache License 2.0
+    that can be found in the LICENSE file.
+   #}
+   {# doc - application traffic recognition #}
+{% if application_traffic_recognition is arista.avd.defined %}
+   !
+
+### Application Traffic Recognition
+
+#### Application Traffic Recognition Summary
+
+##### Categories
+{%     for category in application_traffic_recognition.categories | arista.avd.natural_sort('name') %}
+{%         if category.name is arista.avd.defined %}
+
+###### Category {{category.name}}
```

Comment:
please run `j2lint` (`pre-commit run --all j2lint`) to validate the structure of your template